.github/workflows/dev-long-tests.yml

```diff
@@ -28,7 +28,7 @@ jobs:
     steps:
     - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # tag=v4.1.1
     - name: make test
-      run: make test
+      run: MOREFLAGS="-DZSTD_WINDOW_ALLOW_PICKING_FRACTIONAL_SIZES=1" make test
```

Comment:
I initially thought this compilation variable was enabling or disabling some tests specific to fractional sizes.
But looking at the code below, it rather seems to impact the behavior of the new `windowLogAndFrac` functions.
In which case, I wonder if this is really necessary.
Why would it be useful to (silently) ignore the fractional parameter if a compile-time variable is not set ?